routes/index.js

```diff
@@ -35,6 +35,28 @@ const routes = [
     method: 'GET',
     handler: require('../handlers/search'),
   },
+
+  {
+    path: '/problems/{id?}',
```

Comment:
i copied from this route path: '/trials/{id?}',
I think, id is required here too, right?